src/DataManagement/Saving/AlternativeSavingCalculator.cs

```diff
@@ -37,14 +36,7 @@ public CalculationResults GetSuggestedExpensesOffers(IEntryList entryList, IGoal
             for (var i = enumCount; i > (int)Importance.Necessary; i--)
             {
                 IEntryList expenses;
-                if (!generateRandomData)
-                {
-                    expenses = entryList.GetBy((Importance)i);
-                }
-                else
-                {
-                    expenses = expensesRandomList.GetBy((Importance)i);
-                }
+                expenses = !generateRandomData ? entryList.GetBy((Importance)i) : expensesRandomList.GetBy((Importance)i);
```

Comment:
And further on you can do this 


                var expenses = !generateRandomData ? entryList.GetBy((Importance)i) : expensesRandomList.GetBy((Importance)i);